app/src/main/java/io/intrepid/contest/screens/contestcreation/NewContestActivity.java

```diff
@@ -34,7 +31,8 @@
 import static io.intrepid.contest.screens.contestcreation.addcategoriestocontest.AddCategoryActivity.NOTIFY_NEW_CATEGORY;
 import static io.intrepid.contest.screens.contestcreation.addcategoriestocontest.AddCategoryActivity.makeIntent;
 
-public class NewContestActivity extends BaseMvpActivity<NewContestPresenter> implements NewContestMvpContract.View, EditContestContract {
+public class NewContestActivity extends BaseMvpActivity<NewContestPresenter> implements
+        NewContestMvpContract.View, EditContestContract, ViewPager.OnPageChangeListener {
```

Comment:
your presenter should be the one implementing the `onPageChangeListener` and deciding what fragment to show